### PR TITLE
Removed rabbitmq:glance-simplestreams-sync relation

### PR DESCRIPTION
**Pull request by DavidSpek**
This relation is not supported anymore in OpenStack 20.08
https://docs.openstack.org/charm-guide/latest/2008.html

### CODE REVIEW

File: stable/openstack-base/openstack-base-TPG-monitoring-overlay.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
+---
```
Comment (AurelienLourot) [inline stable/openstack-base/openstack-base-TPG-monitoring-overlay.yaml:1]: Why is this file called ...overlay.yaml? Is this an overlay? Should it be in the overlay folder then?

**Review by AurelienLourot — CHANGES_REQUESTED** [review]

File: stable/openstack-base/openstack-base-TPG-monitoring-overlay.yaml (hunk 1 above)
Comment (DavidSpek) [reply stable/openstack-base/openstack-base-TPG-monitoring-overlay.yaml:1]: Sorry about that, the monitoring overlay I created accidentally got added to this pull request as well. I will move it to the correct folder. Other than that does it look alright?

Comment (AurelienLourot) [reply stable/openstack-base/openstack-base-TPG-monitoring-overlay.yaml:1]: yes, thank you!

Comment (DavidSpek) [reply stable/openstack-base/openstack-base-TPG-monitoring-overlay.yaml]: Should be good now. I noticed charmed kubernetes uses `monitoring-pgt-overlay.yaml` so I named it `openstack-base-monitoring-pgt-overlay.yaml`.

File: stable/openstack-base/openstack-base-monitoring-pgt-overlay.yaml [hunk 2]
```diff
@@ -0,0 +1 @@
+../overlays/openstack-base-monitoring-pgt-overlay.yaml
```
Comment (AurelienLourot) [inline stable/openstack-base/openstack-base-monitoring-pgt-overlay.yaml:1]: I don't think this overlay is core/fundamental enough to deserve a symlink here. Other IMHO more important overlays like Octavia don't have a symlink here.

Apart from that I think the whole patch now looks good, thanks :)

**Review by AurelienLourot — CHANGES_REQUESTED** [review]

**Review by AurelienLourot — APPROVED** [review]

**Pull request merged**